File: RQ2_Prompt_Elements/OpenAI_Data/HumanEvalPython_output/scenario3/id_35T2000Test.py
# id_35_Test.java
from scenario3.id_35 import max_element

import pytest

class id_35_Test:
    """
    Test class of id_35.py.
    It contains ten unit test cases for the :func:`scenario3.id_35.max_element`.
    """


    def test_max_element_1(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3]) == 3

    def test_max_element_2(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([5, 3, -5, 2, -3, 3, 9, 0, 123, 1, -10]) == 123

    def test_max_element_3(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_4(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == -1

    def test_max_element_5(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_6(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_7(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_8(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_9(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_10(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

# id_35_Test.py
from scenario3.id_35 import max_element

import pytest

class id_35_Test:
    """
    Test class of id_35.py.
    It contains ten unit test cases for the :func:`scenario3.id_35.max_element`.
    """
		
    def test_max_element_1(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3]) == 3

    def test_max_element_2(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([5, 3, -5, 2, -3, 3, 9, 0, 123, 1, -10]) == 123

    def test_max_element_3(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_4(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == -1

    def test_max_element_5(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_6(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_7(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_8(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

    def test_max_element_9(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10]) == 10

    def test_max_element_10(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([-1, -2, -3, -4, -5, -6, -7, -8, -9, -10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) == 10

# id_35_Test.java
from scenario3.id_35 import max_element

import pytest

class id_35_Test:
    """
    Test class of id_35.py.
    It contains ten unit test cases for the :func:`scenario3.id_35.max_element`.
    """
		
    def test_max_element_1(self):
        """
        Test case for the :func:`scenario3.id_35.max_element`.
        """
        assert max_element([1, 2, 3]) == 3

    def test_max_element_2(self):
        """

File: RQ2_Prompt_Elements/StarCoder_Data/HumanEvalJava_output/scenario1/id_35T2000Test.java
// MaxElementTest.java
package scenario1;

import java.util.List;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
* Test class of {@link MaxElement}.
* It contains ten unit test cases for the {@link MaxElement#maxElement(List)} method.
*/
class MaxElementTest {

    @Test
    void testMaxElement() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
        assertEquals(10, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement2() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11);
        assertEquals(11, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement3() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12);
        assertEquals(12, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement4() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13);
        assertEquals(13, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement5() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14);
        assertEquals(14, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement6() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
        assertEquals(15, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement7() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16);
        assertEquals(16, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement8() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17);
        assertEquals(17, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement9() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18);
        assertEquals(18, MaxElement.maxElement(l));
    }

    @Test
    void testMaxElement10() {
        List<Integer> l = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19);
        assertEquals(19, MaxElement.maxElement(l));
    }
}
